src/mainwindow/infomarkseditdlg.h

```diff
@@ -44,39 +44,41 @@ class InfoMarksEditDlg : public QDialog, private Ui::InfoMarksEditDlg
 public slots:
 	void objectListCurrentIndexChanged(const QString&);
 	void objectTypeCurrentIndexChanged(const QString&);
-	 
+  void objectClassCurrentIndexChanged(const QString&);
```

Comment:
Eh, the code style in MMapper2 is completely fubar. My best guess is that the official [Qt Code Style](https://wiki.qt.io/Qt_Coding_Style) is probably most fitting. Any thoughts on this?